### ISSUE TITLE
Samples with multiple lane reads

### ISSUE THREAD

**Issue by jb23590** [faircloth-lab/phyluce#351]
Dear Faircloth Lab,

I have a small issue regarding the index sequences. For a few samples I have two different forward and two different reverse reads due to multiple lanes being used.

In such cases, how do I please code this when listing the index sequences per sample so that all are accounted for?

Thank you
Jay

**Comment by jb23590** [comment]
Thank you Brant

Just to confirm, if doing the separate processing route - I should retain these separate files through the illumiprocessor and QC steps and only merge prior to using SPAdes?

**Comment by brantfaircloth** [comment]
that's correct.  you'll need to merge them back together and then place them in the directory structure that's expected (e.g. so the directory/folder structure looks the same as it does for other individuals).
